autoload/vundle/installer.vim

```diff
@@ -6,8 +6,10 @@
 " ...    -- any number of bundle specifications (separate arguments)
 " ---------------------------------------------------------------------------
 func! vundle#installer#new(bang, ...) abort
-  let bundles = (a:1 == '') ?
+  let bundles = (a:0 == 0) ?
         \ g:bundles :
+        \ (a:bang) ?
```

Comment:
I agree.
But as it's quite straightforward, I thought I should continue with the original style of the expression.
No problems breaking it.